dev/java-code-format-template.xml

```diff
@@ -34,8 +34,8 @@
   <option name="IMPORT_LAYOUT_TABLE">
     <value>
       <emptyLine />
-      <package name="javax" withSubpackages="true" static="false" />
       <package name="java" withSubpackages="true" static="false" />
+      <package name="javax" withSubpackages="true" static="false" />
```

Comment:
@chenliang613 @QiangCai Qiang Cai told me that it is wrong. So I change it  by the way